packages/core/src/sessionManagement.ts

```diff
@@ -65,7 +68,7 @@ export function startSessionManagement<Type extends string>(
 
   return {
     getId() {
-      return sessionId.get()
+      return cookiesAuthorized ? sessionId.get() : undefined
```

Comment:
why adding this logic in core session management? 
this logic should  only apply for logs, right?